x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/grok.tsx

```diff
@@ -129,7 +129,11 @@ export const Grok: FunctionComponent = () => {
           return (
             <DragAndDropTextList
               label={fieldsConfig.patterns.label!}
-              helpText={fieldsConfig.patterns.helpText}
+              helpText={
```

Comment:
this is follow-up to https://github.com/elastic/kibana/pull/192083
I missed it in the pr